src/cephadm/cephadm.py

```diff
@@ -3872,7 +3872,6 @@ def get_unit_file(ctx, fsid):
 ExecStart=/bin/bash {data_dir}/{fsid}/%i/unit.run
 ExecStop=-/bin/bash -c 'bash {data_dir}/{fsid}/%i/unit.stop'
 ExecStopPost=-/bin/bash {data_dir}/{fsid}/%i/unit.poststop
-KillMode=none
```

Comment:
I don't think this is a good idea. We did set killMode=none for very good reasons. at least back then, there was a race between systemd and podman and very rarely systemd killed processes earlier than podman resulting in a broken state and podman being unable to properly shut down a container. killMode=none prevented this race. I'd be super careful with this.

Reply:
Right, See https://github.com/ceph/ceph/pull/33162#issuecomment-584183316 .

Reply:
Thanks for the feedback! I wondered if this might be dangerous. I will mark this as a draft and dig deeper into if (and why) processes might not be shutting down properly.